src/index.js

```diff
@@ -1,34 +1,39 @@
+import MyTodoClass from './modules/application.js';
 import './style.css';
 
-document.addEventListener('DOMContentLoaded', () => {
-  const todoItems = [
-    {
-      description: 'Do homework',
-      completed: true,
-      index: 0,
-    },
-    {
-      description: 'Go shopping',
-      completed: true,
-      index: 1,
-    },
-    {
-      description: 'Prepare a budget',
-      completed: true,
-      index: 2,
-    }];
-  todoItems.forEach((todoItems) => {
-    const display = document.querySelector('.display');
-    const items = document.createElement('div');
-    items.innerHTML = `
-    <form action="/action_page.php">
-  
-  <div class ="push">
-  <input type="checkbox" id="stuff" name="stuff">
-  <label for="inputs" class="inputs"> ${todoItems.description}</label>  <i class="fa-solid fa-ellipsis-vertical"></i>  
-  </div>
-  </form>`;
+const input = document.querySelector('#run');
+const form = document.querySelector('.form');
+const button = document.querySelector('#check');
 
-    display.appendChild(items);
+MyTodoClass.display();
+
+form.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const desc = input.value;
+  const done = new MyTodoClass(desc);
+  done.addtodolist();
+});
```

Comment:
- [ ] Currently i can add an empty todo, Kindly fix this 
![Screenshot from 2022-07-07 13-53-11](https://user-images.githubusercontent.com/66097371/177778743-382b8718-8add-450d-8764-703a608edf96.png)

Reply:
Thanks will make the change